src/pages/StatusPage.vue

```diff
@@ -19,7 +19,10 @@
                 <!-- Description -->
                 <div class="my-3">
                     <label for="description" class="form-label">{{ $t("Description") }}</label>
-                    <textarea id="description" v-model="config.description" class="form-control" data-testid="description-input"></textarea>
+                    <textarea
```

Comment:
please remove all accidental changes to this file.
It is a large enough PR as is.